src/test/kotlin/com/ampnet/userservice/grpc/GrpcUserServerTest.kt

```diff
@@ -217,6 +218,35 @@ class GrpcUserServerTest : TestBase() {
         }
     }
 
+    @Test
+    fun mustReturnAllUsersForCoop() {
+        suppose("Coop exists") {
+            testContext.coop = createCoopResponse(COOP)
+            Mockito.`when`(coopService.getCoopByIdentifier(testContext.coop.identifier)).thenReturn(testContext.coop)
```

Comment:
Suggestion: switch to BDD Mockito - then you would have `given(someMock.someCall()).willReturn(someValue)`. For assertions you can then use `then(mock).should(times(1)).methodCall()`.